src/components/node-list/index.js

```diff
@@ -134,18 +136,23 @@ const NodeListProvider = ({
   });
 
   return (
-    <NodeList
-      items={items}
-      sections={sections}
-      groups={groups}
-      searchValue={searchValue}
-      onUpdateSearchValue={updateSearchValue}
-      onToggleGroupChecked={onToggleGroupChecked}
-      onItemClick={onItemClick}
-      onItemMouseEnter={onItemMouseEnter}
-      onItemMouseLeave={onItemMouseLeave}
-      onItemChange={onItemChange}
-    />
+    <div
+      className={classnames('pipeline-nodelist', {
+        'pipeline-nodelist--fade': isFaded
```

Comment:
I think this might best moved into `node-list.js` directly and given a prop, leaving the data responsibility here in this provider but toggling via the prop?

Reply:
Sure. I hate passing props down through files but it's no big deal. Done